### DISCUSSION TITLE
Multiple TV Show Libraries Selector/Option

### DISCUSSION THREAD

**Discussion by Healzangels** [DouwJacobs/plex-shuffler#113] [Ideas]
Greetings!
Thanks for creating and sharing such an awesome tool, it's be great to use!
I was wondering if it would be possible to include the ability to change between different Libraries designated as "TV Shows"

I have an Anime TV Library as well as a TV Shows Library which both show under the "Default TV Show Library*" selector. 

Ideally it would be awesome if possible to swap between the different libraries or even have a merged view to be able to create a mixed content shuffle with both "tv shows" and anime or be able to create both tv playlists as well as anime playlists from their given libraries.

Maybe this is already possible, and I've totally overlooked it as well! 😁 

-Cheers!

**Comment by DouwJacobs** [comment]
Good morning @Healzangels !

Currently it is possible to switch between libraries by going to Settings -> General (Tab) -> Default TV Show Library

I'll have a look and see how it will work to combine multiple libraries or just add a drop down to switch between libraries on the fly 😁

**Reply by Healzangels** [reply]
That'd be awesome!

I have a few friends who have access to my Plex server so while navigating to setting and swapping the default tv show library around for myself has worked out well, unfortunately doesn't for them. Thanks!

**Reply by DouwJacobs** [reply]
Hi @Healzangels!

Users can now set their default library. You can test it using the latest development image. 

User Profile -> Settings -> Playlists -> User Default TV Show Library


Will also check to add it on the homepage after some testing.

Thanks for the suggestion 😊

**Comment by Healzangels** [comment]
Fantastic! Really appreciate all your time on this, works great! Thanks again for sharing such an awesome tool and can't wait for future updates!
